pkg/client/parse.go

```diff
@@ -141,12 +144,6 @@ func parseTestSuite(testSuite junit.TestSuite, tags string, verbose bool) (suite
 			return
 		}
 
-		if verbose {
```

Comment:
i would prefer we keep this unlesss we log this somewhere else ?

Reply:
Condensed all SpecRun verbose logging into line 160